src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -477,7 +477,7 @@ const requestToState = (composeRequest, distroInfo, isProd, enableOscap) => {
 
     // oscap policy
```

Comment:
The last "policy" left behind. It sneaked back with the conflict.

Reply:
All fixed now :slightly_smiling_face: